site/content/en/docs/adoption/_index.md

```diff
@@ -29,6 +29,8 @@ aliases:
 
 **Nvidia**: LeaderWorkerSet deployments are the recommended method for deploying Multi-Node models with NIM, see document [here](https://docs.nvidia.com/nim/large-language-models/1.5.0/deploy-helm.html#multi-node-models).
 
+**Red Hat**: LeaderWorkerSet is available in Red Hat OpenShift by installing the Leader Worker Set Operator from OperatorHub, see document [here](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/ai_workloads/index#leader-worker-set-operator).
```

Comment:
```suggestion
**Red Hat**: LeaderWorkerSet is available in Red Hat OpenShift by installing the LeaderWorkerSet Operator from OperatorHub, see document [here](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/ai_workloads/index#leader-worker-set-operator).
```

Reply:
That is a good point @bergerhoffer. We need to align with our official product name. I reverted back to original state.